frameworks/Rust/ntex/benchmark_config.json

```diff
@@ -0,0 +1,61 @@
+{
+  "framework": "ntex",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "Postgres",
+      "framework": "ntex",
+      "language": "Rust",
+      "orm": "Raw",
+      "platform": "None",
+      "webserver": "ntex",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "ntex",
+      "notes": "",
+      "versus": ""
+    },
+    "db": {
+      "fortune_url": "/f",
+      "db_url": "/d",
+      "query_url": "/q?q=",
+      "update_url": "/u?q=",
```

Comment:
Also /fortune, /db, /query and /update.

Reply:
this should be reflected in ticket

Reply:
@nbrady-techempower are those mandatory paths as well?

Reply:
@fafhrd91 Sorry, yes. I should have made that clearer. The verification changes in #5648 account for minimum required lengths for all routes.

Reply:
sure, np. Updated